scripts/InstallHootDevelopmentEnvironment.sh

```diff
@@ -43,7 +43,7 @@ fi
 # use tomcat 7 here instead of tomcat 6 (what's tested with hoot...I don't really like this...but), since eclipse 
 # installed from repos (installed in this script later) needs tomcat 7
```

Comment:
Could this file somehow be combined with the vagrant provision script? Seems like there is a lot of overlap.